fapi-client/src/main/scala/com/gu/facia/api/contentapi/ContentApi.scala

```diff
@@ -99,7 +99,7 @@ object ContentApi extends StrictLogging {
   def parseQueryString(queryString: String): Seq[(String, String)] = {
     val KeyValuePair = """([^=]+)=(.*)""".r
 
-    queryString split "&" collect {
+    queryString.split("&").toIndexedSeq.collect {
```

Comment:
Can just do `toSeq` - don't think the performance characteristics of `IndexedSeq` are required?

```suggestion
    queryString.split("&").toSeq.collect {
```